workflow_deeplabcut/pipeline.py

```diff
@@ -0,0 +1,46 @@
+import datajoint as dj
+from element_lab import lab
+from element_animal import subject
+from element_session import session_with_datetime as session
+from element_deeplabcut import train, model
+
+from element_animal.subject import Subject
+from element_session.session_with_datetime import Session
```

Comment:
You can replace this line with `Session = session.Session` somewhere further down